spec/support/with_json_schema_context.rb

```diff
@@ -19,6 +19,8 @@ def load_json_schema(file_name)
   end
 end
 
-RSpec.configure do |rspec|
-  rspec.include_context 'with json schema', with_json_schema: true
+RSpec.configure do |config|
+  config.before(:suite) do
+    config.include_context 'with json schema'
+  end
 end
```

Comment:
as discussed it looks like this only affect the 
`with json schema` (being inside the file with_json_schema_context.rb )